singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -236,8 +254,14 @@ class SpinerEOSDependsRhoSie : public EosBase<SpinerEOSDependsRhoSie> {
     printf("%s\n\t%s\n\t%s%i\n", s1, s2, s3, matid_);
     return;
   }
+  PORTABLE_INLINE_FUNCTION const Transformer &getTransformer() const {
+    return transformer_;
+  } // getter for tranformation structs
```

Comment:
We may want to rename this to something more descriptive, like `getSieTransformer` or something similar.